packages/ruleset/src/rules/no-ambiguous-paths.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+const { oas3 } = require('@stoplight/spectral-formats');
+const { noAmbiguousPaths } = require('../functions');
+
+module.exports = {
+  description: 'Avoid ambiguous path strings within an OpenAPI document',
+  message: '{{error}}',
+  severity: 'warn',
+  formats: [oas3],
+  resolved: true,
+  given: ['$'],
```

Comment:
Is there a reason not to use `$.paths` here? I know it doesn't really make a difference, just curious

Reply:
I initially tried that, but ran into some sort of weird spectral problem.  I don't remember exactly what I did to fix that, but I don't think the "given" value was the cause.  I'll change it back.

Reply:
fixed